docs/sources/setup-grafana/configure-security/configure-authentication/saml-ui/index.md

```diff
@@ -0,0 +1,52 @@
+
+- Knowledge of SAML authentication. Refer to [SAML authentication in Grafana]({{< relref "../saml/" >}}) for an overview of Grafana's SAML integration.
+- Permissions that allow you to read and update authentication settings. These permissions are granted by `fixed:authentication.config:writer` role.
+  By default, this role is granted to Grafana server administrator in self-hosted instances and to Organization admins in hosted Grafana instances.
+- Grafana instance running Grafana version 10.0 or later with [Grafana Enterprise]({{< relref "../../../introduction/grafana-enterprise/" >}}) or [Grafana Cloud Pro or Advanced](/docs/grafana-cloud/) license.
+
+## Steps
+
+Follow these steps to configure and enable SAML integration:
+
+1. Sign in to Grafana and navigate to **Administration > Authentication > Configure SAML**.
+1. Complete the **General settings** section. Consult the following table for additional guidance about certain fields:
+
+| Field               | Description                                                                                                                                                                                                                                             |
+| ------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| signup              | Determines whether to allow new Grafana user creation through SAML login. If disabled, then only existing Grafana users can log in with SAML.                                                                                                           |
+| single logout       | SAML's single logout feature allows users to log out from all applications associated with the current IdP session established via SAML SSO. Consult [SAML single logout documentation]]({{< relref "../saml/#single-logout" >}}) for more information. |
+| IdP initiated login | Allows users to log into Grafana directly from the SAML identity provider (IdP). Consult [IdP initiated login documentation]({{< relref "../saml/#idp-initiated-single-sign-on-sso" >}}) for more information.                                          |
+
+1. Click **Next: Key and certificate**.
+1. Provide a certificate and a private key that will be used by the service provider (Grafana) and the SAML identity provider. The private key needs to be issued in the [PKCS #8](https://en.wikipedia.org/wiki/PKCS_8) format. You can refer to our SAML documentation for [an example on how to generate SAML credentials]({{< relref "../saml/#example-of-how-to-generate-saml-credentials" >}}).
+1. Specify whether you want the outgoing requests to be signed, and, if so, which signature algorithm should be used.
+1. Click **Next: Connect Grafana with Identity Provider** and complete the section.
```

Comment:
I wonder if I can see the UI. I'm curious about this flow and the UI labels.

Reply:
Sure! We can click through the UI once you're back from the break to see how it looks like.

Reply:
Should we mention that signing requests is recommended since without it you're vulnerable to MITM attack?